js/main.js

```diff
@@ -206,8 +221,19 @@ function initMarker(feature) {
         openingHoursUnclassified = properties.opening_hours_unclassified;
     } else {
         var openingRanges = getOpeningRanges(openingHoursStrings);
-        todayOpeningRange = getOpeningRangeForDate(openingRanges, now);
-        timeTableHtml = getTimeTable(openingRanges);
+        /* If no opening hours or a next date, don't show a marker. */
+        if (openingRanges === null) {
+            return;
+        }
+        /* Are there opening hours in the current week? */
+        else if ('intervals' in openingRanges) {
```

Comment:
Can you please use `openingRanges.hasOwnProperty('intervals')` instead of this. This will also return true if `intervals` is found somewhere along the prototype chain as far as I know.